### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by the-lazy-fox**
Proposition to improve the README. To be replicated into Build for Windows FAQ.

### CODE REVIEW

**Comment by ugd** [comment]
I confirm, the new readme works for me!

File: README.md [hunk 1]
```diff
@@ -50,22 +58,27 @@ If any changes are made to `jellyfin-web/`, the `jellyfin-web/dist/` directory w
 
 ```sh
 cd jellyfin-tizen
-JELLYFIN_WEB_DIR=../jellyfin-web/dist yarn install
+JELLYFIN_WEB_DIR=../jellyfin-web/dist 
+yarn install
```
Comment (T-bond) [inline README.md:62]: Why this change? How will yarn now about the environment variable then?

**Review by T-bond — COMMENTED** [review]
It replaces all Linux commands with Windows ones.
Also it breaks one of the command with the environment variable seperation.

So I don't see why this would be an improvement.

**Review by dmitrylyzo — COMMENTED** [review]
I started reviewing the PR some time ago :sweat_smile:  and got stuck on this:
README.md is mainly for Linux. For Windows I made Wiki page, but it is not so easy to contribute to it.
Currently, I am thinking of making an extra `master` branch (`wiki-master`) in this repo, which will replicate the Wiki repo (me or bot will push changes to it).
This will allow everyone to open PRs on that branch as usual.

But if anyone has another idea, I'm all ears.

**Comment by dmitrylyzo** [comment]
@the-lazy-fox I created a Wiki master branch: `wiki/master`. So Wiki PRs can be targeted on it.

**Comment by the-lazy-fox** [comment]
They work perfectly for me and are much more detailed. Honestly, it was a pain to follow the Readme on Windows when you never did any of this.
Feel free to reuse them in any other guide you will build.
